pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TableViewTest.java

```diff
@@ -222,7 +222,6 @@ public void testPublishNullValue() throws Exception {
         assertEquals(tv.get("key2"), "value2");
     }
 
-
```

Comment:
We don't need the `testAck` method in #17728 ?

Reply:
I see an internal data structure change for the `reader` member in TableView. The `testAck` in the master branch will fail in 2.10 branch.

2.10
`private final ConcurrentMap<String, Reader<T>> readers;`

master
`private final CompletableFuture<Reader<T>> reader;`

Reply:
> We could add the test when merging this update in branch-2.10.

I see, it's introduced in https://github.com/apache/pulsar/pull/15589, which is a refactor and not necessary for patch releases.